packages/python/plotly/plotly/tests/test_optional/test_utils/test_utils.py

```diff
@@ -195,9 +210,9 @@ def test_figure_json_encoding(self):
         _json.dumps(figure, cls=utils.PlotlyJSONEncoder, sort_keys=True)
 
         # Test data wasn't mutated
-        np_array = np.array([1, 2, 3, np.NaN, np.NAN, np.Inf, dt(2014, 1, 5)])
+        np_array = np.array([1, 2, 3, np_nan(), np_inf(), dt(2014, 1, 5)])
         for k in range(len(np_array)):
-            if k in [3, 4]:
+            if k == 3:
```

Comment:
How about the case of 4?

Reply:
This just removes the second nan in the list, so it's only at index 3.